common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -527,12 +541,14 @@ class @Problem
         answered = false
       if bind
         $(select_field).on 'change', (e) =>
-          @submitAnswersAndSubmitButton()
+          @submitAnswersAndSubmitButton false, true
           return
         return
 
     if answered
       @enableSubmitButton true
+      if data_changed
+        @saveNotification.remove()
```

Comment:
Is there a need to only do this conditionally? Can we just not remove a save notification (if it exists) whenever the submit button is re-enabled?